sr_unix/ydbenv.mpt

```diff
@@ -98,7 +98,7 @@ set	; Set up environment (environment variables, database, etc.) with default st
 	set tmp2=$zsearch(tmp1_"/libyottadbutil.so",$increment(zstmp))
 	set:'$zlength(tmp2) tmp2=tmp2=$zsearch(tmp1_"/libgtmutil.so",$increment(zstmp))
```

Comment:
is there a reason why tmp2=tmp2=$zsearch? From my testing (though on a broken system of sorts) this is what adds the 1 I mentioned earlier.

Reply:
Good catch. I will fix and resubmit.